lib/utils/job-utils/ipmitool.js

```diff
@@ -163,16 +162,44 @@ function ipmitoolFactory(Promise) {
     };
 
     /**
-     * Returns a promise with the results or errors of invoking sel list -c
+     *  Returns a promise with the results or errors of invoking sel list -c
      *
      * @param host
      * @param user
      * @param password
      * @param count
      */
-    Ipmitool.prototype.sel = function(host, user, password, count) {
+    Ipmitool.prototype.sel = function(host, user, password, count) {
         return this.runCommand(host, user, password, "-c sel list last " + count);
     };
+    
+    /**
+     * Returns a promise with the results or errors of invoking sel get <entry> ...<entry>
+     *
+     * @param host
+     * @param user
+     * @param password
+     * @param Sel Entries
+     */
+    Ipmitool.prototype.selEntry = function(host, user, password, stringEntries) {
+        return this.runCommand(host, user, password, "sel get " + stringEntries);
+    };
+
+    /**
+     * Returns a promise with the results or errors of invoking sel raw
+     *
+     * @param host
+     * @param user
+     * @param password
+     * @param Sel Entries
+     */
+    Ipmitool.prototype.readRawSel = function(host, user, password, selEntry) {
+        var tmp = selEntry.split("") ;
+        var command = "raw 0x0a 0x43 0x00 0x00 " + 
```

Comment:
@tannoa2 : I don't think it is self-explanation enough, not every developer is familiar with IPMI and let alone the raw command.  Even add something like "NetFn=0x0a, Command=0x43 is Get SEL Entry" will provide a lot clue if other developer want to lookup the detail command format.